### PR TITLE
documentation: update tag cloud/filtering doc

### PR DESCRIPTION
**Pull request by nodiscc**
Ref. https://github.com/shaarli/Shaarli/issues/959

### CODE REVIEW

File: doc/md/Browsing-and-searching.md [hunk 1]
```diff
@@ -14,10 +14,24 @@ Use the `Filter by tags` field to restrict displayed links to entries tagged wit
 
 **Hidden tags:** Tags starting with a dot `.` (example `.secret`) are private. They can only be seen and searched when logged in.
 
-Alternatively you can use the `Tag cloud` to discover all tags and click on any of them to display related links.
+### Tag cloud
 
-To search for links that are not tagged, enter `""` in the tag search field.
+The `Tag cloud` page diplays a "cloud" view of all tags in your Shaarli.
+
+ * More frequently used tags are displayed with a bigger font size.
```
Comment (virtualtam) [inline doc/md/Browsing-and-searching.md]: s/More/The most/

File: doc/md/Browsing-and-searching.md [hunk 2]
```diff
@@ -14,10 +14,24 @@ Use the `Filter by tags` field to restrict displayed links to entries tagged wit
...
+ * When sorting by `Most used` or `Alphabetical`, tags are displayed as a _list_, along with counters and edit/delete button for each tag.
```
Comment (virtualtam) [inline doc/md/Browsing-and-searching.md]: button**s**

File: doc/md/Browsing-and-searching.md [hunk 3]
```diff
@@ -14,10 +14,24 @@ Use the `Filter by tags` field to restrict displayed links to entries tagged wit
...
+ * Clicking on any tag will display a list of all Shaares matching this tag.
+ * Clicking on the counter next to a tag `example` , will filter the tag cloud to only display tags found in Shaares tagged `example`. Repeat this any number of times to further filter the tag cloud. Click `List all links with those tags` to display Shaares matching your current tag filter.
```
Comment (virtualtam) [inline doc/md/Browsing-and-searching.md]: no space before comma

**Comment by nodiscc** [comment]
Merged into https://github.com/shaarli/Shaarli/pull/1036

**Pull request closed without merging**